CompetitionTemplate/src/competition/subsystems/arm/arm_commands/ArmToTopCommand.java

```diff
@@ -0,0 +1,40 @@
+package competition.subsystems.arm.arm_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import competition.subsystems.arm.ArmTargetSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ArmToTopCommand extends BaseCommand {
+    ArmSubsystem armSubsystem;
+    ArmTargetSubsystem armTargetSubsystem;
+
+    @Inject
+    public ArmToTopCommand(ArmSubsystem armSubsystem, ArmTargetSubsystem armTargetSubsystem) {
+        this.armSubsystem = armSubsystem;
+        this.armTargetSubsystem = armTargetSubsystem;
+        this.requires(this.armSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        armTargetSubsystem.setTargetAngle(90);
+    }
+
+    @Override
+    public void execute() {
+        
+    }
+    
+    @Override
+    public boolean isFinished() {
+        return armSubsystem.getArmAngle() >= armTargetSubsystem.getTargetAngle();
```

Comment:
This time the sign is right, but we need to be a little more flexible about knowing if we're at our goal yet. Maybe look at the absolute difference and see if that's larger than some factor (and make that a double property so we can easily change it). Also 90 should be a double property since we'll want to tweak that as well.